.github/workflows/ci-sbt.yml

```diff
@@ -41,11 +41,12 @@ jobs:
           distribution: 'corretto'
           cache: 'sbt'
 
-      # (Respects .nvmrc)
       - name: Setup Node
-        uses: guardian/actions-setup-node@v2.4.1
```

Comment:
This action has been deprecated, in favour of the first party `actions/setup-node`.